site/en/docs/extensions/reference/declarativeNetRequest/extra_permissions.md

```diff
@@ -0,0 +1,3 @@
+Hello, I'm an extra *permission*
+
+Am I [linked](https://google.com) to stuff?
```

Comment:
Why is this here?

Reply:
This was a first attempt at including Markdown directly in another page. I forgot to remove it!